raiden.spec

```diff
@@ -107,7 +107,7 @@ else:
         a.scripts,
         exclude_binaries=True,
         name=executable_name,
-        debug=False,
+        debug=True,
```

Comment:
Hmm ... ?

Reply:
This is only used when the env variable `ONEFILE` is set to `0` to produce an exploded directory instead of a single packed executable.

We're (well I guess so far only I and maybe Paul ;)) only using that mode for debugging purposes, so having `debug=True` for PyInstaller makes sense.